portal-impl/src/com/liferay/portal/util/PropsValues.java

```diff
@@ -1895,6 +1895,9 @@
 	public static final String NETVIBES_SERVLET_MAPPING = PropsUtil.get(
 		PropsKeys.NETVIBES_SERVLET_MAPPING);
 
+	public static final String NODEJS_NODE_ENV = GetterUtil.getString(
+		PropsUtil.get(PropsKeys.NODEJS_NODE_ENV));
+
```

Comment:
Apparently, it wasn't exposed but I'm not sure if this was intentional, let me know if there is any problem.

Reply:
This is a build-time property which is in default [build.properties](https://github.com/liferay/liferay-portal/blob/master/build.properties#L509-L511) file. 

The original requirement was set at build time, meaning the tool sources would only package a bundle with the checker if one builds the portal in development mode.

This would prevent our tool from being released.

Reply:
Hmm Okay, I think I get it, in this case, I'm enabling in runtime instead of build time.